creation/web_base/condor_platform_select.sh

```diff
@@ -18,15 +18,15 @@
 glidein_config="$1"
 tmp_fname="${glidein_config}.$$.tmp"
 
-error_gen="`grep '^ERROR_GEN_PATH ' "$glidein_config" | cut -d ' ' -f 2-`"
+# import add_config_line function
+add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | cut -d ' ' -f 2-)
+source "$add_config_line_source"
```

Comment:
Add the shellcheck line:
```
# shellcheck source=./add_config_line.source
```
prefer `.` to `source`

Reply:
updated